NMS-13330: Add out of band monitoring content to main user documentation (#3453)
OpenNMS/opennms: docs/modules/reference/pages/configuration/minion-confd/jmx-exporter.adoc

[[jmx-exporter]]
= Prometheus JMX exporter

> To provide an out-of-band management of the JVM with the Minion process, this container image ships with the Prometheus JMX exporter.

Reviewer: Is this "out-of-band management" or "out-of-band monitoring"?  The former implies the Prometheus JMX Exporter would offer remote management, not just a way to get performance metrics.
```suggestion
To provide out-of-band management of the JVM for the Minion process, the Minion container image ships with the Prometheus JMX exporter.
```

Author: > Is this "out-of-band management" or "out-of-band monitoring"? The former implies the Prometheus JMX Exporter would offer remote management, not just a way to get performance metrics.

I'm not sure. The original text said management, but I imagine monitoring is correct. I'll verify.

Author: Monitoring it is!
